Kentico/KInspector
# Use Action to generate build artifact

### Motivation

Removes appveyor references and uses the existing GitHub action to generate a release package for users.

### Checklist


### How to test

1. Check GH action is passing from latest commit, or push a commit with minor change to re-run action
2. Download "KInspector" artifact from passing action
3. Extract .zip archive locally
4. Run `KenticoInspector.WebApplication.exe` and ensure application runs without errors
